zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -56,7 +56,8 @@ pub(crate) const MAINNET_ACTIVATION_HEIGHTS: &[(block::Height, NetworkUpgrade)]
     (block::Height(653_600), Blossom),
     (block::Height(903_000), Heartwood),
     (block::Height(1_046_400), Canopy),
-    // TODO: Add Nu5 mainnet activation height
+    // TODO: Add the correct Nu5 mainnet activation height
+    (block::Height(10_000_000), Nu5),
```

Comment:
I had to put something here so i can make the tests. Same as the testnet Nu5 height. Open to other number suggestions.

Reply:
Ah, that makes sense. I had to do the same for some NU5 tests too. But before merging I was planning on removing the height and add `#[should_panic]` to the tests that require the height (and also a `TODO` to remove them later). @teor2345 had also mentioned that it's a good idea to list the tests that we do this in #1841, so we can remove the `#[should_panic]`s and `TODO`s when the height is set :smile:

Reply:
The Nu5 activation heights were removed.